Duplicati/Library/Utility/FilterGroups.cs

```diff
@@ -556,8 +556,10 @@ private static IEnumerable<string> CreateLinuxFilters(FilterGroup group)
             }
             if (group.HasFlag(FilterGroup.CacheFiles))
             {
+                yield return FilterGroups.CreateWildcardFilter(@"*/.cache/");                
                 yield return FilterGroups.CreateWildcardFilter(@"*/.config/google-chrome/Default/Cookies");
-                yield return FilterGroups.CreateWildcardFilter(@"*/.config/google-chrome/Default/Cookies-journal");
+                yield return FilterGroups.CreateWildcardFilter(@"*/.config/google-chrome/Default/Cookies-journal");                
+

```

Comment:
Can we remove this blank line?

Reply:
Yes you can remove it, or I have to?

Reply:
It would be nice if you can remove it.  When possible, we would prefer to clean up the changes before merging them so that we don't have to remember to do so later.

Reply:
Sorry create a new Pull request.  I'm new with git.  New PR #3713